augur/filter/filter_and_subsample.py

```diff
@@ -752,6 +750,8 @@ def apply_filters(metadata, exclude_by, include_by):
 
     return strains_to_keep, strains_to_filter, strains_to_force_include
 
+=======
+>>>>>>> 65217e87 (augur.filter.rules: Move filtering rules and related functions here)
```

Comment:
Looks like some merge conflicts snuck into this commit.

Reply:
😭 I was banging my head yesterday over merge conflicts, guess I didn't bang hard enough. Let me fix that

Reply:
Done – it was in 7df06b88202f63b47b1c098c59843c9efb329f76 which is now f88d3d472dcc307a974aa99ba7d978020f355a67.